app/controllers/business_support_controller.rb

```diff
@@ -38,7 +38,10 @@ def send_slimmer_headers
 
   def prepare_facets
     @facets = {}
-    @facets[:postcode] = params[:postcode] if params[:postcode].present?
+    # Strip all non-alphanumeric characters apart from whitespace from the postcode.
+    if params[:postcode].present?
+      @facets[:postcode] = params[:postcode].gsub(/[^\w\s\d]/i, '').strip
```

Comment:
[`\w` means `[a-zA-Z0-9_]`](http://ruby-doc.org/core-2.1.5/Regexp.html#class-Regexp-label-Character+Classes) so you don't also need the `\d` in here.